scripts/west_commands/run_common.py

```diff
@@ -511,9 +511,7 @@ def rebuild(command, build_dir, args):
 def runners_yaml_path(build_dir, board):
     ret = Path(build_dir) / 'zephyr' / 'runners.yaml'
     if not ret.is_file():
-        log.die(f'either a pristine build is needed, or board {board} '
-                "doesn't support west flash/debug "
-                '(no ZEPHYR_RUNNERS_YAML in CMake cache)')
+        log.die(f'no runners.yaml found in {build_dir}/zephyr. A pristine build might be needed')
```

Comment:
I think the detail about not supporting west flash etc. should stay. `runners.yaml` will not be generated when building for certain boards (such as `mps2`), which is normal, and a pristine build won't change that.

Reply:
I wasn't aware of that and I agree, thanks. I have updated the PR.

Reply:
Great, now could you rebase your PR on the up-to-date `main` and fix the merge conflict?